frontend/src/scenes/plugins/edit/PluginSourceTabs.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { pluginSourceLogic } from 'scenes/plugins/edit/pluginSourceLogic'
+import { LemonButton } from 'lib/components/LemonButton'
+import { LemonRow } from 'lib/components/LemonRow'
+
+export function PluginSourceTabs(): JSX.Element {
+    const { setCurrentFile } = useActions(pluginSourceLogic)
+    const { currentFile, fileNames, pluginSourceValidationErrors } = useValues(pluginSourceLogic)
+
+    return (
+        <LemonRow style={{ padding: 0 }}>
+            {fileNames.map((fileName) => (
+                <LemonButton
+                    key={fileName}
+                    type={currentFile === fileName ? 'secondary' : 'tertiary'}
```

Comment:
These types don't perform super well here:
- `tertiary` is intended for really "extra" actions, which we don't want to clutter up the screen too much for users (like "Show details" on insight cards) – but tabs are a key element of navigation, so we definitely want those buttons to look clickable
- the `secondary` button is outlined, which gives it a ton of individuality, making the pattern feel less like tabs and more like disparate actions
I suggest using the default type for cohesiveness and instead marking the active file with the `active` prop.